app/interactors/send_fresh_sheet.rb

```diff
@@ -22,6 +22,11 @@ def send_test_email
   end
 
   def send_fresh_sheets_to_subscribed_members
+    unless market.active?
+      context[:notice] = "Provided market (#{market.name}) for fresh sheet distribution is not active."
```

Comment:
this is the actual fix - we never actually checked to see if a given market was active or not.

Reply:
TODO - remove market active check (and spec)